index.js

```diff
@@ -23,5 +26,24 @@ module.exports.sync = (fp, data) => {
 
 	fp = path.basename(fp) === 'package.json' ? fp : path.join(fp, 'package.json');
 
+	data = normalize(data);
+
 	writeJsonFile.sync(fp, data, opts);
 };
+
+function normalize(pkg) {
+	const npkg = {};
+	for (const key of Object.keys(pkg)) {
+		switch (key) {
```

Comment:
Would look better to just use a `Set` with the dependency keys and then check it in the loop: `npkg[key] = dependencyKeys.has(key) ? sortKeys(...) : pkg[key];`